core/domain/exp_domain.py

```diff
@@ -1396,6 +1393,16 @@ def update_interaction_fallbacks(self, fallbacks_list):
         self.interaction.fallbacks = [
             Fallback.from_dict(fallback_dict)
             for fallback_dict in fallbacks_list]
+        if self.interaction.fallbacks:
+            hint_list = []
+            for fallback in self.interaction.fallbacks:
+                if fallback.outcome.feedback:
+                    # If a fallback outcome has a non-empty feedback list
+                    # the feedback is converted to a Hint. It may contain
+                    # only one list item.
```

Comment:
I'm not sure there can be more than one feedback. I couldn't find a way to add multiple feedbacks for the same fallback in the Editor UI.